pkgs/applications/editors/edit/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, fetchgit, unzip, pkgconfig, ncurses, x11, cwebbin }:
```

Comment:
@joachifm updated. However, I have a question. Why does `x11` work? I cant see that package in the `top-level/all-packages.nix`?